.github/workflows/triage.yml

```
@@ -20,42 +19,105 @@ jobs:
             --raw-field query='
-              query($owner: String!, $repository_name: String!, $project_number: Int!) {
-                repository(owner: $owner, name: $repository_name) {
-                  project(number: $project_number) {
-                    columns(first: 10) {
+              query($organization: String!, $project_number: Int!) {
+                organization(login: $organization){
+                  projectV2(number: $project_number) {
+                    id
+                    fields(first:20) {
                       nodes {
-                        id,
-                        name
+                        ... on ProjectV2Field {
+                          id
+                          name
+                        }
+                        ... on ProjectV2SingleSelectField {
+                          id
+                          name
+                          options {
+                            id
+                            name
+                          }
+                        }
                       }
                     }
                   }
                 }
               }' > project_columns.json
-          echo 'COLUMN_ID='$(jq '.data.repository.project.columns.nodes[] | select(.name == "Triage") | .id' project_columns.json) >> $GITHUB_ENV
-          echo 'COLUMN_NAME='$(jq '.data.repository.project.columns.nodes[] | select(.name == "Triage") | .name' project_columns.json) >> $GITHUB_ENV
+          echo 'PROJECT_ID='$(
+            jq \
+              '.data.organization.projectV2.id' \
+              project_columns.json
+          ) >> $GITHUB_ENV
+          echo 'STATUS_FIELD_ID='$(
+            jq \
+              '.data.organization.projectV2.fields.nodes[]
+              | select(.name== "Status")
+              | .id' \
+              project_columns.json
+          ) >> $GITHUB_ENV
+          echo 'TRIAGE_COLUMN_ID='$(
+            jq --raw-output \
+              '.data.organization.projectV2.fields.nodes[]
+              | select(.name== "Status")
+              | .options[]
+              | select(.name=="To Triage")
+              | .id' \
+              project_columns.json
+          ) >> $GITHUB_ENV
 
-      - name: Add issue#${{ github.event.issue.number }} to Triage column
+      - name: Add issue#${{ github.event.issue.number }} to project
         env:
           ISSUE_ID: ${{ github.event.issue.node_id }}
         run: |
-          echo "Adding issue: ${{ github.event.issue.number }} to column $COLUMN_NAME in project $PROJECT_NUMBER"
-          if [[ $DRY_RUN == "false" ]]; then
+          echo "Adding issue: ${{ github.event.issue.number }} to project $PROJECT_NUMBER"
+          if [[ $DRY_RUN == 'false' ]]; then
+            item_id="$(
+              gh api graphql \
+                --jq '.data.addProjectV2ItemById.item.id' \
+                --raw-field project_id=$PROJECT_ID \
+                --raw-field issue_id=$ISSUE_ID \
```

These probably won't contain whitespace but I'd quote just in case anyway:
```suggestion
                --raw-field project_id="$PROJECT_ID" \
                --raw-field issue_id="$ISSUE_ID" \
```

Same in other commands.